Update preparations.md

Katjadoerle opened the pull request.

Katjadoerle committed:
Update preparations.md

das-g commented on preparations.md:
```diff
@@ -1,32 +1,32 @@
 > **Achtung:** Wenn du OS X 10.6, 10.7, oder 10.8 hast, musst du die Version für OS X Snow Leopard auswählen!
 
-> macOS High Sierra: Es ist nicht mehr möglich, die vorab installierte (aber nicht brauchbare) git-Version von OSX mittels "sudo mv /usr/bin/git /usr/bin/git-system" einfach umzubenennen, wie von git-scm vorgeschlagen. Der einfachste Weg, deine neue git-Installation zum Laufen zu bringen, ist, eine neue Datei in deinem Home-Ordner zu erstellen, mit dem Namen .bash_profile und nur einer Zeile: 'ex ...
+> macOS High Sierra: Es ist nicht mehr möglich, die vorab installierte (aber nicht brauchbare) git-Version von OSX mittels "sudo mv /usr/bin/git /usr/bin/git-system" einfach umzubenennen, wie von git-scm vorgeschlagen. Der einfachste Weg, deine neue git-Installation zum Laufen zu bringen, ist, eine neue Datei in deinem Home-Ordner zu erstellen, mit dem Namen .bash_profile und nur einer Zeile: 'ex ...
```
Hier ist [Terminal-Gerät in der EDV](https://www.duden.de/rechtschreibung/Terminal_Geraet) (immer neutrum) gemeint, nicht [Terminal-Gebäude/-Anlage am Flughafen oder Bahnhof](https://www.duden.de/rechtschreibung/Terminal_Empfangsgebaeude_Anlage) (maskulin oder neutrum möglich). Muss daher "dein" bleiben.

das-g commented on preparations.md:
```diff
@@ -1,32 +1,32 @@
 #### Git installieren
 
-Git ist ein Versionskontrollsystem, das von vielen Programmiern verwendet wird. Die Software behält den Überblick über Änderungen an Dateien, so dass man frühere Versionen jederzeit wiederherstellen kann. Ein bisschen wie die 'Änderungen Aufzeichnen'-Funktione von Microsoft Word, aber viel mächtiger.
+Git ist ein Versionskontrollsystem, das von vielen Programmierern verwendet wird. Die Software behält den Überblick über Änderungen an Dateien, so dass man frühere Versionen jederzeit wiederherstellen kann. Ein bisschen wie die 'Änderungen Aufzeichnen'-Funktion von Microsoft Word, aber viel mächtiger.
```
:+1:

das-g commented on preparations.md:
```diff
@@ -41,9 +41,9 @@ Gehe auf [GitHub.com](https://GitHub.com) und erstelle dir einen neuen Gratisacc
 
 ### Die Einleitung lesen
-Der letzte Schritt der Vorbereitung ist dir das Kapitel ['How the Internet Works'](https://tutorial.djangogirls.org/en/how_the_internet_works/) vom Django Girls Tutorial durchzulesen.
+Der letzte Schritt zur Vorbereitung besteht darin, dir das Kapitel ['How the Internet Works'](https://tutorial.djangogirls.org/en/how_the_internet_works/) im Django Girls Tutorial durchzulesen.
```
:+1:

Zusätzlich könnte man noch das "dir" entfernen. "Durchlesen" kann man zwar reflexiv verwenden, muss hier aber nicht.

das-g committed:
Belasse "Terminal" neutrum

Hier ist Terminal-Gerät in der EDV (immer neutrum) gemeint, nicht Terminal-Gebäude/-Anlage am Flughafen oder Bahnhof (maskulin oder neutrum möglich). Muss daher "dein" bleiben.

das-g committed:
Entferne unnötiges "dir"

"Durchlesen" kann man zwar reflexiv verwenden, muss hier aber nicht.

das-g committed:
Merge branch 'master' into patch-4
